src/Components/Medicine/PrintPreview.tsx

```diff
@@ -42,61 +42,62 @@ export default function PrescriptionsPrintPreview() {
 
   return (
     <PrintPreview
-      title={
-        patient ? `Prescriptions - ${patient.name}` : "Print Prescriptions"
-      }
+      title={patient ? `Prescriptions - ${patient.name}` : "Print Prescriptions"}
       disabled={!(patient && encounter && items)}
     >
       <div className="mb-3 flex items-center justify-between p-4">
         <h3>{encounter?.facility_name}</h3>
         <img className="h-10 w-auto" src={main_logo.dark} alt="care logo" />
       </div>
-      <div className="mb-6 grid grid-cols-8 gap-y-1.5 border-2 border-secondary-400 p-2">
-        <PatientDetail name="Patient" className="col-span-5">
-          {patient && (
-            <>
-              <span className="uppercase">{patient.name}</span> -{" "}
-              {t(`GENDER__${patient.gender}`)},{" "}
-              {patientAgeInYears(patient).toString()}yrs
-            </>
-          )}
-        </PatientDetail>
-        <PatientDetail name="IP/OP No." className="col-span-3">
-          {encounter?.patient_no}
-        </PatientDetail>
+      <div className="mb-6 grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-y-1.5 border-2 border-secondary-400 p-2">
+  <PatientDetail name="Patient" className="col-span-1 md:col-span-2 lg:col-span-2">
+    {patient && (
+      <>
+        <span className="uppercase">{patient.name}</span> -{" "}
+        {t(`GENDER__${patient.gender}`)},{" "}
+        {patientAgeInYears(patient).toString()}yrs
+      </>
+    )}
+  </PatientDetail>
+  <PatientDetail name="IP/OP No." className="col-span-1 md:col-span-1 lg:col-span-1">
+    {encounter?.patient_no}
+  </PatientDetail>
 
-        <PatientDetail
-          name={
-            encounter
-              ? `${t(`encounter_suggestion__${encounter.suggestion}`)} on`
-              : ""
-          }
-          className="col-span-5"
-        >
-          {formatDate(encounter?.encounter_date)}
-        </PatientDetail>
-        <PatientDetail name="Bed" className="col-span-3">
-          {encounter?.current_bed?.bed_object.location_object?.name}
-          {" - "}
-          {encounter?.current_bed?.bed_object.name}
-        </PatientDetail>
+  <PatientDetail
+    name={
+      encounter
+        ? `${t(`encounter_suggestion__${encounter.suggestion}`)} on`
+        : ""
+    }
+    className="col-span-1 md:col-span-2 lg:col-span-2"
+  >
+    {formatDate(encounter?.encounter_date)}
+  </PatientDetail>
+  <PatientDetail name="Bed" className="col-span-1 md:col-span-1 lg:col-span-1">
+    {encounter?.current_bed?.bed_object.location_object?.name}
+    {" - "}
+    {encounter?.current_bed?.bed_object.name}
+  </PatientDetail>
+
+  <PatientDetail name="Allergy to medication" className="col-span-1 md:col-span-2 lg:col-span-4">
+    {patient?.allergies ?? "None"}
+  </PatientDetail>
```

Comment:
@ayushpatil2122 Like already mentioned in the issue, let's not make the print preview child responsive based on screen size. This would mean, the print output could differ on different screen size than what was shown in the preview.

Reply:
sorry i will fix that